src/appshell/qml/NotationPage/NotationPage.qml

```diff
@@ -337,10 +349,6 @@ DockPage {
                 Component.onCompleted: {
                     mixerPanel.contextMenuModel = contextMenuModel
                 }
-
-                onResizeRequested: function(newWidth, newHeight) {
-                    mixerPanel.resize(newWidth, newHeight)
-                }
```

Comment:
This code should not be removed. It makes sure that the Mixer automatically adjusts its size when rows (e.g. effects field, volume field, aux sends) are shown/hidden (if you click the '...' button and then in the View submenu).

Reply:
So can I make an if inside the function in which it first checks if the new width is smaller than the minimum and if it isn't, it doesn't resize to the width?

Reply:
That would be an option. But I would try first whether it already works correctly without an `if`. I would expect that the resize function checks internally whether the new size is valid compared to the min/max size. But if it really doesn't work without an `if` you can certainly add one.